plaidml2/bridge/keras/__init__.py

```diff
@@ -929,7 +1131,10 @@ def random_uniform(shape, minval=0.0, maxval=1.0, dtype=None, seed=None):
 
 
 def random_uniform_variable(shape, low, high, dtype=None, name=None, seed=None):
-    _report_unimplemented('random_uniform_variable')
+    if seed:
+        np.random.seed(seed)
```

Comment:
It'd be really nice to avoid these legacy global-state-polluting methods, but I'd worry about back-compat.  :-/